notebooks/WET/WET-003/WET-003-ComCam.ipynb

```diff
@@ -7,217 +7,757 @@
    "source": [
```

Comment:
<p>closest  / furthest to the center of FOV ? (I guess)   - because one could also think of a center of each CCD, but I think that's not it given you're looking at vignetting here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>